lib/carrierwave/sequel.rb

```diff
@@ -11,6 +11,43 @@ def mount_uploader(column, uploader, options={}, &block)
       raise "You need to use Sequel 3.0 or higher. Please upgrade." unless ::Sequel::Model.respond_to?(:plugin)
       super
 
+      mod = Module.new
+      include mod
+      mod.class_eval <<-RUBY, __FILE__, __LINE__+1
+        def #{column}=(new_file)
+          if !(new_file.blank? && send(:#{column}).blank?)
+            modified!
+          end
+          super
+        end
+
+        def remove_#{column}!
```

Comment:
I just went off this https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/orm/activerecord.rb#L84